SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -155,6 +268,12 @@ bool FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 		return false;
 	}
 
+	if (!PreStartCheck())
```

Comment:
Let's change the function name to something more verbose, such as `LocalDeploymentPreRunChecks`